src/django_components/slots.py

```diff
@@ -1102,7 +1102,11 @@ def render_func(ctx: Context, slot_data: Dict[str, Any], slot_ref: SlotRef) -> S
 
         trace_component_msg("RENDER_NODELIST", component_name, component_id=None, slot_name=slot_name)
 
-        rendered = template.render(ctx)
+        # We wrap the slot nodelist in Template. However, we also override Django's `Template.render()`
+        # to call `render_dependencies()` on the results. So we need to set the strategy to `raw`
+        # so that the dependencies are processed only once the whole component tree is rendered.
+        with ctx.push({"DJC_DEPS_STRATEGY": "raw"}):
```

Comment:
Had to wrap slots in `DJC_DEPS_STRATEGY="raw"`, because we render slots via `Template.render()`.